deployments.json

```diff
@@ -25,6 +28,9 @@
     },
     "BulkIssue": {
       "startBlock": 12378987
+    },
+    "BulkTransfer": {
+      "startBlock": 11907787
```

Comment:
here too (see polygon section comment) this should be at least as height as the other contracts but ideally set to the height the contract is deployed at.